Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }

```

Comment:
**Suggestion**: Rename the method from "Even" to "printEvenNumbers" to adhere to Java naming conventions and clarify the method's purpose. Refactor the method to improve code reusability and performance by using a modular approach and optimizing the loop condition to avoid potential ArrayIndexOutOfBoundsException. <br> **Code Suggestion**: 
 ```
 -    public static void Even(String[] args) {
 +    public static void printEvenNumbers(int[] arr) {
 ```